src/controllers/app_controller.go

```diff
@@ -0,0 +1,113 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+	"golang-bootcamp-2020/services"
+	_errors "golang-bootcamp-2020/utils/error"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type AppController interface {
+	FetchData(c *gin.Context)
+	GetHealth(c *gin.Context)
+	GetCharacterById(c *gin.Context)
+	GetCharacters(c *gin.Context)
+	GetCharacterIdByName(c *gin.Context)
+}
+
+type appController struct {
+	service services.Service
+}
+
+type idResponse struct {
+	Id string `json:"id"`
+}
+
+//Return new pointer to application controller
+func NewAppController(service services.Service) AppController {
+	return &appController{service}
+}
+
+//Return ok if api works correctly
+func (ac *appController) GetHealth(c *gin.Context) {
+	c.String(http.StatusOK, "ok")
+}
+
+//Fetch data from rick and morty api
+func (ac *appController) FetchData(c *gin.Context) {
+
```

Comment:
By convention, you shouldn't start a code block with an empty line
Please remove all the empty lines at the beginning of the functions